packages/backend/src/tokens/machine.ts

```diff
@@ -2,7 +2,7 @@ import type { AuthenticateRequestOptions } from '../tokens/types';
 import type { MachineTokenType } from './tokenTypes';
 import { TokenType } from './tokenTypes';
 
-export const M2M_TOKEN_PREFIX = 'mt_';
+export const M2M_TOKEN_PREFIX = 'm2m_';
```

Comment:
missed this, but luckily the early access partners are using api_keys and oauth_token for now